Assets/HoloToolkit/Utilities/Scripts/HeadsetAdjustment.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using HoloToolkit.Unity.InputModule;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class HeadsetAdjustment : MonoBehaviour, IInputClickHandler, ISpeechHandler {
+
+    [Tooltip("The name of the scene to load when the user is ready. If left empty, the next scene is loaded as specified in the 'Scenes in Build')")]
+    public string NextSceneName;
```

Comment:
There's a way to reference the scene object in the inspector instead of a string.